resources/web/docs_js/index.js

```diff
@@ -267,5 +267,23 @@ $(function() {
     document.head.appendChild(hotcss);
   }
 
+  // If you're have been given access to the private repository these docs
+  // relate to *and* you use the 'edit' in the query string or hash, we'll show
+  // you an edit button taking you to the private repo.
```

Comment:
The comment is not correct - this code doesn't test if you have access to the private repo.

```suggestion
  // For the private docs repositories, the edit button is hidden
  // unless there is an '?edit' in the query string or hash.
```